Projects/Checkers/Board.cs

```diff
@@ -0,0 +1,171 @@
+﻿namespace Checkers;
+
+public class Board
+{
+	public List<Piece> Pieces { get; set; }
+
+	public Piece? Aggressor { get; set; }
+
+	public Piece? this[int x, int y] =>
+		Pieces.FirstOrDefault(piece => piece.X == x && piece.Y == y);
+
+	public Board()
+	{
+		Aggressor = null;
+		Pieces = new List<Piece>
+			{
+				new() { NotationPosition ="A3", Color = Black},
```

Comment:
Keep pondering this. Piece constructors could be more DRY re color but it would probably harm readability as well as custom set up so maybe a bit of a bagatelle.

Reply:
Feel free to open up a new pull request in the future if you think of something you prefer.